imap_processing/spice/geometry.py

```diff
@@ -209,6 +209,10 @@ def frame_transform(
                     f"{np.asarray(et).size} elements."
                 )
 
+    # If from_frame and to_frame are the same, no rotation needed
```

Comment:
Do we even need to do all of the checks above in this caes?

Reply:
Good point - done